{{cookiecutter.project_slug}}/README.md

```diff
@@ -2,7 +2,31 @@
 
 This is a repository for a web application developed with Django, built with [Crowdbotics](https://crowdbotics.com)
 
-### Features
+## Table of Contents
+
+1. [Project Structure](#project-structure)
+2. [Features](#features)
+3. [Getting Started: Backend](#getting-started-backend)
+   - [Docker Setup (recommended)](#docker-setup-recommended)
+   - [Local Setup](#local-setup-alternative-to-docker)
+4. [Usage](#usage)
+   - [Admin Panel](#admin-panel)
+   - [API Documentation](#api-documentation)
+
+## Project Structure
+
+    ..
+    ├── home                     # Starter home app
+    ├── modules                  # Crowdbotics Modules app
+    ├── plat_5213_dev_22643      # Django project configurations
+    ├── static                   # Static assets
+    ├── users                    # Starter users app
+    ├── web_build                # React Native Web build
+    ├── ...
+    ├── README.md
+    └── ...
+
+## Features
 
 1. **Local Authentication** using email and password with [allauth](https://pypi.org/project/django-allauth/)
 2. **Rest API** using [django rest framework](http://www.django-rest-framework.org/)
```

Comment:
line 34 _4. Bootstrap4_ should not have a link attached to the bootstrap4 docs?

Reply:
Good suggestion. I took these as it is from the previous docs so I'll add the links.